calculator/js/script.js

```diff
@@ -0,0 +1,35 @@
+let output = document.querySelector('.output');
+
+let mathEquation = ''
+
+const buttons = document.querySelectorAll('.number');
```

Comment:
@paulghaddad Originally, I had two css classes on several html elements, but ran into issues when I tried applying JS to only of the two properties. Is it a best practice to ONLY use one class per element when working with JS?